packages/iTwinUI-react/src/core/Toast/Toaster.test.tsx

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+import { act } from 'react-dom/test-utils';
```

Comment:
Why imported not from testing-library?

Reply:
I'm just gonna quote this
https://github.com/iTwin/iTwinUI-react/pull/623#discussion_r871357029

But removed all use cases of that import..